.github/workflows/package-ci.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will run eslint for the package
+
+name: Package CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lintSrc:
+    runs-on: ubuntu-latest    
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+          with:
+            node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn run eslint --ext .js --ext .jsx ./src
```

Comment:
You should add: `"scripts": { "eslint" : "eslint --ext .js --ext .jsx ./src" } }` in packages.json and put here `yarn eslint`.

This way the command for CI will be the same that the command on our laptop.